app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
I'll quickly mock this up here in the PR so you can see what I mean. Hope this is all coming in the way it was intended. I would love to contribute to this as I think getting Redux into Ember and making it easy for folks is pretty important.